src/components/Image/Image.tsx

```diff
@@ -112,6 +121,7 @@ export class Image extends React.Component<ImageProps> {
             )}
             <MediaImage
               {...imageProps}
+              {...dimensions}
```

Comment:
Notice that at the moment we support only 1:1 aspect ratio, which means we actually cannot reflect the `contain` option for a relative media item. In the next task we'll support different aspect ratios and then `contain` will be also reflected for media items. 

However, for pre-made (absolute) URLs with specified dimensions that are different from the provided `width` & `height` it will work starting this PR.